### ISSUE TITLE
Windows compatible SF mono regular is missing

### ISSUE THREAD

**Issue by lukilester** [thelioncape/San-Francisco-family#3]
SF-Mono-Regular in the "Windows compatible variants" subfolder is missing

**Comment by thelioncape** [comment]
I know this has been open for a while but I've had a number of commitments and this takes a back seat...

Any idea how to create them? I barely use Windows anymore so I haven't had the need to consider what's needed for the variants or even what's actually different.

Bit of history: this repo was a fork of another that has since been deleted so not much of this is my own compilation

**Comment by jdraggy** [comment]
I have installed the normal Regular variation and this is the result:
![Image](https://github.com/user-attachments/assets/1d9b9e18-fc7a-4cf0-8b47-938269536889)

**Comment by kenmcd** [comment]
The original **SF Mono Regular** font does not require fixing for Windows.
So it should just be copied into the `Windows compatible variants` folder.

**Comment by jdraggy** [comment]
But I saw that the original SF Mono's character map is not the same as the Windows compatible variants.

**Comment by kenmcd** [comment]
> But I saw that the original SF Mono's character map is not the same as the Windows compatible variants.

When the other fonts were modified, two unnecessary `cmap` tables were added.
They are not in the original fonts, and obviously Apple does not need them.
The `cmap` table format which is there, is the same one used by Microsoft.
So no changes were necessary to the `cmap` table for Windows compatibility.
Whatever app was used to fix the fonts probably added them.

In these older SF Pro fonts Apple purposely sabotages the style-linking which is used in Windows applications such as Word, LibreOffice, etc. Affects Linux apps too.
So the compatibility fixes are mostly in the `name` table.
Basically making them "compatible" is fixing what Apple broke.

Should not have any problems using the original Regular font.
It is OK as-is.

**Comment by jdraggy** [comment]
Character map of the SF Pro fonts, Windows converted, is different than SF Pro Regular in the macOS version.

<img width="1600" height="234" alt="SF Mono Regular" src="https://github.com/user-attachments/assets/adeafce9-4f38-428d-bce7-733081026450" />

<img width="1599" height="233" alt="SF Mono Heavy, Wc version" src="https://github.com/user-attachments/assets/0c455798-1809-46d9-bb18-458e0e74733b" />

**Comment by jdraggy** [comment]
Also the Windows compatible versions of SF Pro (~100 KB) seems smaller in size in comparison to the macOS version (~3 MB)
